### ISSUE TITLE
Error getting stream / new pc version ?

### ISSUE THREAD

**Issue by woowoots** [alamminsalo/orion#283]
Hello, since a long time now i'am on the 1.6.1 version, everything was working correctly until today, when, when i changed stream suddently no way to have any stream work.

And its only a orion issue, since without it, i have no problem to watch a stream.

Tried to reboot my computer, still nothing.

Anything to do about that ? And is there a new / update orion pc version aviable ?

thanks !

**Comment by rakslice** [comment]
Seems like there is a breaking API change or a temporary problem? I'm seeing this too. 

Every channel gives e.g. 
`Error transferring https://api.twitch.tv/api/channels/foo/access_token - server replied: Gone`

**Comment by rakslice** [comment]
Streamlink saw some rumblings on this API a couple of days ago... https://github.com/streamlink/streamlink/issues/2680

**Comment by rakslice** [comment]
I reproduced that the payload it's getting with this `410 Gone` is indeed `{"error":"Gone","status":410,"message":"this API has been removed."}`
 but the `https://api.twitch.tv/api/channels/foo/access_token` endpoint is still what the website is using...

**Comment by woowoots** [comment]
dont know... tried the orion 1.6.6 ( forgot i had the link for the lastest version aviable in .exe )
And same issue, cant get stream work, despite seem streamer i follow online, i can see the "thumbmail" and so...

Note that, since a couple of weeks maybe, there was time to time issue for orion to display the game list ( but if i were manualy search for a game, it was working )

So.... 

If i use cherryplayers for exemple, no issue to get a stream ( even tho i would clearly perfer still use orion ^^ )

**Comment by rakslice** [comment]
FYI this access_token API of Twitch is part of how Orion gets access to the video stream; if it's gone there is simply no way to get video

**Comment by rakslice** [comment]
I'm not going to chase unscrupulous ppl into the territory of ripping off a client id from the Twitch website or some other 3rd party app, so as far as I'm concerned the next step is that Twitch starts allowing access to this API again or provides a different one or otherwise issues a statement

**Comment by rakslice** [comment]
And maybe they never issue a statement https://discuss.dev.twitch.tv/t/410-gone-when-requesting-access-token-json-api-with-3rd-party-client-id/23060/4

**Comment by rakslice** [comment]
I tried again just now and the API seems to be working again. No substantive response to on the dev forum thread, and nothing really new from other open 3rd party client issues other than them also having noticed it's working again.

**Comment by rakslice** [comment]
... though I should say that the moderator on the Twitch dev site did come back and more thoroughly spell out the finger wagging that they expect 3rd parties not to be using undocumented APIs, which is at least some kind of a response.

If they were doing timed outages as part of a staged sunset, without saying anything publicly, outages for 15 minutes then an hour then six hours might be consistent with that, in which case the moderator response today will at least in hindsight have been a warning that the API could disappear altogether.

I'm not sure why they would do a staged sunset of this API, as it's not like the newer API suites have any replacement for it as far as I can tell.

**Comment by woowoots** [comment]
Hello, yeah it work again correctly now ^^ nice that was just a twitch API thing issue, and not a complet change that make orion unusable :)

Btw, for me the 1.6.6 version crash when i open a stream, but the 1.6.1 correctly work

**Comment by rakslice** [comment]
I created a request for this on Twitch's dev uservoice https://twitch.uservoice.com/forums/310213-developers/suggestions/38997565-documented-api-for-streams-and-vods

**Comment by rakslice** [comment]
It's failing again right now.

**Comment by woowoots** [comment]
yeah, i cannot use Orion once again ^^

**Comment by CuddleBear92** [comment]
Still down btw. Other apps is also having issues.

https://twitter.com/backseatgamer_/status/1194038946898096130?s=20

A change in the code is prob needed as they needed to patch theirs aswell.

**Comment by woowoots** [comment]
yeah, that's down again after a brief come back, its annoying what they are doing with their API :( 

cherryplayers still work, but well, since cant log in with that 3rd party, its not that great ( and it use nearly as much ressources as using a browser to watch twitch, so well

**Comment by Pauloo27** [comment]
I'm having `Token issue. Error getting stream` on Fedora Linux (orion built from source), is that the same issue?

**Comment by woowoots** [comment]
i suppose yes that's the same issue because of an twitch API change

any words of if an update to have orion work again will be done ? :o

**Comment by thaewrapt** [comment]
Some insight on a possible workaround from the Streamlink issue linked earlier:
https://github.com/streamlink/streamlink/issues/2680#issuecomment-557605851

**Comment by bisam** [comment]
The pull request fixes streaming, but logging in is not possible anymore... At least according to thie streamlink comment.. Damn
